### PR TITLE
Fixed attribute value prefix with dynamic content being ignored

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue - #2547 
@NTaylorMullen @dougbu

### CODE REVIEW

File: test/Microsoft.AspNet.Mvc.FunctionalTests/BasicTests.cs [hunk 1]
```diff
@@ -77,6 +77,24 @@ public class BasicTests
         }
 
         [Fact]
+        public async Task ViewWithAttributePrefix_RendersWithoutIgnoringPrefix()
+        {
+            // Arrange
+            var server = TestHelper.CreateServer(_app, SiteName, _configureServices);
+            var client = server.CreateClient();
+            var expectedContent = await _resourcesAssembly.ReadResourceAsStringAsync("compiler/resources/BasicWebSite.Home.ViewWithPrefixedAttributeValue.html");
+
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/BasicTests.cs]: empty line

File: test/WebSites/BasicWebSite/Views/Home/ViewWithPrefixedAttributeValue.cshtml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿@{
+    Layout = "/Views/Shared/_Layout.cshtml";
```
Comment (NTaylorMullen) [inline test/WebSites/BasicWebSite/Views/Home/ViewWithPrefixedAttributeValue.cshtml]: In this test, no reason to have a layout or `ViewBag.Title`, should be very precise.

**Comment by NTaylorMullen** [comment]
:watch:

File: test/Microsoft.AspNet.Mvc.FunctionalTests/BasicTests.cs (hunk 1 above)
Comment (dougbu) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/BasicTests.cs]: must have been intended to hold the extra characters on the previous line :smiling_imp:

**Comment by dougbu** [comment]
:shipit: when @NTaylorMullen is satisfied

**Comment by ajaybhargavb** [comment]
Updated.

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**